posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,268 @@
+blog_description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this beta release, the features that make up the new Jakarta EE 10 Core Profile are made available: 
+
+* Jakarta RESTful Web Services 3.1: support multipart/form-data.
+* Jakarta JSON-B 3.0: deserialization of null to JsonValue.NULL, default PropertyVisibilityStrategy Implemenation etc.
+* Jakarta JSON-P 2.1: handling duplicated keys; defined more interfaces for primitives, lists and objects 
+* Jakarta CDI 4.0: defining new extension APIs; removing reflections; adding new reflection-free language model; empty beans.xml means that only annotated classes are beans; removal of deprecated APIs
+** Jakarta Annotation 2.1: allowing `@Priority` to be used on any program elements; adding `@Nullable` and `@NotNull`
+** Jakarta Interceptors 2.1: picking up the changes from Annotation 2.1
+
+In addition to the Core Profile features, Jakarta EE 10 Expression Language 5.0 is also included in this beta and provides a number of new features and specification clarifications.   
+
+But it's not just Jakarta features that are making their debut in this release!  MicroProfile OpenAPI has been updated to version 3.1, and with it provides the ability to specify optional authentication for operations and multiple annotation enhancements.  Last but not least, Password Utilities 1.1 is introduced in this beta, and with it the ability to no longer forcefully federate stand-alone user registries.
+
+
+The link:{url-about}[Open Liberty] 22.0.0.9-beta includes the following beta features (along with all GA features):
+
+* <<restful, RESTful Web Services 3.1>>
+* <<json, Jakarta JSON Binding and Processing>>
+* <<cdi, CDI 4.0>>
+* <<expression, Jakarta EE 10 Expression Language 5.0>>
+* <<microprofile, MicroProfile OpenAPI 3.1>>
+* <<password, Password Utilities 1.1>>
+
```

Comment:
```suggestion
More information about Jakarta EE 10 Core Profile can be found [here](https://jakarta.ee/specifications/coreprofile/10/).
```

Reply:
I don't think we need this since the link was added to the opening paragraph.